caf-audit-maven-plugin/caf-audit-maven-plugin.md

```diff
@@ -49,12 +51,7 @@ provided next.
 </AuditedApplication>
 ```
 
-`AuditedApplication` is the root element. `ApplicationId` identifies the name of the application that the audit event type 
-is associated with. For each audit event defined, `TypeId` is a string identifier for the particular event (e.g. viewDocument) 
-and `CategoryId` is a string identifier for the category of event. A list of parameter elements are then defined for each audit 
-event. This includes the name of the parameter, the type (i.e. string, short, int, long, float, double, boolean or date), 
-description and an optional `ColumnName` element which identifies the database column to be used for storage in the event 
-that multiple audit events comprise of parameters with the same name. 
+`AuditedApplication` is the root element. `ApplicationId` identifies the name of the application that the audit event type is associated with. For each audit event defined, `TypeId` is a string identifier for the particular event (e.g. viewDocument) and `CategoryId` is a string identifier for the category of event. A list of parameter elements are then defined for each Audit Event. This includes the `Name` of the parameter, the `Type` (i.e. string, short, int, long, float, double, boolean or date) and the `Description`. The `IndexingHint` (i.e. fulltext or keyword) is optional and can be used to specify an indexing hint when storing audit event parameter data of `Type` string. The `Constraints` element is also optional and this can be used to specify minimum and/or maximum length constraints for audit event parameters of `Type` string. The `ColumnName` element too is optional which can be used to force the use of a particular database column when storing the audit data. 
```

Comment:
@mulhollandc and @CJComac 

"The `ColumnName` element too is optional which can be used to force the use of a particular database column when storing the audit data."

Is this still valid in today's implementation?  There is no concept of column's in Elasticsearch.